kernel/float64.mli

```diff
@@ -19,8 +19,16 @@ val is_nan : t -> bool
 val is_infinity : t -> bool
 val is_neg_infinity : t -> bool
 
-val to_string : t -> string
 val of_string : string -> t
+(** Print a float exactly as an hexadecimal value (exact decimal
+ * printing would be possible but sometimes requires more than 700
+ * digits). *)
+val to_string : t -> string
+(** Print a float as a decimal value. The printing is not exact (the
```

Comment:
Please add some spacing so we can tell what the comment is attached to

Reply:
done, thanks